mltsp/tests/test_flask_app.py

```diff
@@ -1554,7 +1554,7 @@ def test_prediction_proc(self):
             entry = r.table("predictions").get("TEMP_TEST01").run(conn)
             pred_results_list_dict = entry
             assert(pred_results_list_dict["pred_results_list_dict"]
-                   ["TESTRUN_215153.dat"][0][0] in ['Beta_Lyrae',
+                   ["TESTRUN_215153"][0][0] in ['Beta_Lyrae',
                                                     'Herbig_AEBE'])
```

Comment:
Yeah, I forgot to save my vim plugins from my old machine and I'm still trying to get the indentation working correctly...